encoding/csv_stringify_test.ts

```diff
@@ -481,7 +481,7 @@ Deno.test(
     async fn() {
       const columns = [0];
       const data = [[(n: number) => n]];
-      const output = `0${NEWLINE}(n) => n${NEWLINE}`;
+      const output = `0${NEWLINE}(n)=>n${NEWLINE}`;
```

Comment:
This is a reasonable change. It just emits differently than typescript does. If anything now our emits will be more consistent than before.